Raylib-cs/types/Mesh.cs

```diff
@@ -24,37 +33,37 @@ public unsafe partial struct Mesh
         /// <summary>
         /// Vertex position (XYZ - 3 components per vertex) (shader-location = 0)
         /// </summary>
-        public float* Vertices;
+        public float* Vertices = default;
```

Comment:
Struct field initialization is required for compiling when constructor is declared.